packages/dotcom-ui-header/src/header.scss

```diff
@@ -34,48 +34,9 @@
 .o-header__drawer-search-term {
   width: 100%;
 }
-.o-header__top-wrapper .ft-header__ask-ft-button {
-  padding: 8px;
-}
-.ft-header__ask-ft-button {
-  @include oTypographySans(-1, $weight: 'semibold');
-  color: oColorsByName('ft-grey');
-  background-color: oColorsMix('black', 'paper', 5);
-  border-radius: 4px;
-  padding-right: 8px;
-  text-transform: uppercase;
-  text-decoration: none;
-  white-space: nowrap;
-
-  &::before {
-    content: '';
-    display: block;
-    @include oIconsContent(
-      $icon-name: 'sparkles',
-      $color: oColorsByName('black'),
-      $size: 32
-    );
-    vertical-align: top;
-  }
-
-  .ft-header__ask-ft-button-label {
-    display: inline-block;
-    padding: 8px 0;
-    line-height: 15px;
-  }
-}
 
-.ft-header__top-ask-ft-button {
-  @include oGridRespondTo($until: 'L') {
-    display: none;
+.o-header__ask-ft-button {
+  &, &:hover {
+    color: oColorsByName('black-80');
   }
 }
```

Comment:
**question:** @Ventzy is this still required? Does the Origami styles not include this?

Reply:
Origami `o-header` doesn't need it - colors stays unchanged in its [storybook](https://main--655f72ec522e424302dc6201.chromatic.com/?path=/story/o2-core_components-o-header--header-primary&globals=backgrounds:!undefined). But in `dotcom-ui-header` the color is changed on hover. 
I've just dug into it, and it seems to come through several packages `dotcom-ui-layout/styles` -> `dotcom-ui-base-styles/styles` -> `n-ui-foundations/main` which includes `oTypographyLink()` from `origami/o-typography` for `a` tags. And it has color on hover. I've set it for both states here to ensure sure that nothing will affect it. Probably it will be best to be added in `o-header` directly. Wdyt?

Reply:
To make sure I understand correctly, this is to override a rule set in `n-ui-foundations` that is not present in Origami. If so, this looks good to me! `n-ui-foundations` is a pain 😄 Lets leave this override here. It would be good to add a comment to say this is to override a `n-ui-foundations` css. Would you be okay to do that?

Reply:
The end CSS is in Origami as a mixin and it is included in `n-ui-foundations`.
Defined here https://github.com/Financial-Times/n-ui-foundations/blob/main/typography/main.scss as
```	
        a {
		@include oTypographyLink();
	}
```
And the mixin with the CSS is https://github.com/Financial-Times/origami/blob/main/components/o-typography/src/scss/use-cases/_general.scss#L111

Still good to go as is (with added comment)?

Reply:
Added in https://github.com/Financial-Times/dotcom-page-kit/pull/1052/commits/793f2f6d6c51292d196da776f25ea0ef5570c1f2